ui/pager.go

```diff
@@ -451,3 +464,54 @@ func glamourRender(m pagerModel, markdown string) (string, error) {
 
 	return content.String(), nil
 }
+
+func (m *pagerModel) initWatcher() {
+	var err error
+	m.watcher, err = fsnotify.NewWatcher()
+	if err != nil {
+		log.Error("error creating fsnotify watcher", "error", err)
+	}
+}
+
+func (m *pagerModel) watchFile() tea.Msg {
+	path := m.relFilePath()
+
+	if err := m.watcher.Add(path); err != nil {
+		log.Error("error adding file to fsnotify watcher", "error", err)
+		return nil
+	}
+
+	log.Info("fsnotify watching file", "file", path)
+
+	for {
+		select {
+		case event, ok := <-m.watcher.Events:
+			if !ok || !event.Has(fsnotify.Write) {
+				continue
+			}
+			log.Debug("fsnotify event", "file", path, "event", event.Op)
+			return reloadKeyMsg
```

Comment:
i would probably make a `type reloadMsg struct{}` instead of triggering a key press event

Reply:
Good suggestion. Just pushed.